StitchApp/WorkerPoolStaging/functions/validateUserEntitlements/source.js

```diff
@@ -0,0 +1,29 @@
+exports = async function(payload) {
+  
+  // do some kind of validation
+  var returnObject = { 'valid': true };
+  
+  const db_name = context.values.get('db_name');
+  const coll_name = 'entitlements';
+  const userEmail = payload.email;
+  const repoName = payload.repoName;
+  let userEntitlements = [];
+  
+  // get the queue collection
```

Comment:
We should probably refactor this so that we don't distribute the find logic for entitlements in various places (as it is a mongodb binding). Not an emergency but we should consider it for a refactor eventually.